src/main/java/com/google/ehub/data/EntertainmentItemDatastore.java

```diff
@@ -79,6 +80,7 @@ public Key addItemToDatastore(EntertainmentItem item) {
     itemEntity.setProperty(WRITERS_PROPERTY_KEY, item.getWriters());
     itemEntity.setProperty(ACTORS_PROPERTY_KEY, item.getActors());
     itemEntity.setProperty(OMDB_ID_PROPERTY_KEY, item.getOmdbId());
+    itemEntity.setProperty(NUMBER_OF_LIKES_PROPERTY_KEY, item.getNumberOfLikes());
```

Comment:
I dont think you need to store the number of likes as a field of Entertainment entity. You can get that information directly from the FavoriteItemDatastore.

Reply:
Agreed, better to have one source of truth

Reply:
Also in general (and this is what we do at Google too), if you're making major changes to some part of the project like this, it'd be good to run it by the team first. Otherwise, you run the risk of writing a lot of code and needing to redo it if something comes up

Reply:
But having it as a property makes it possible for me to sort it in Datastore.

Reply:
Ideally you can just sort in memory by grabbing all items and doing your own sort. I'm guessing that breaks pagination?

Reply:
I would have to find a way to link the number of likes to each of the items. I could query for the number of likes for each item but that could get slow if I start having a lot of them.

Reply:
As we discussed in the daily sync, we can continue to work on this PR. The datastore API doesn't provide a way to join queries.